tests/.gitignore

```diff
@@ -0,0 +1,4 @@
+/*.fits
```

Comment:
Are we saving files in the working directory that need to be removed (for example, after running the unit tests)?

Reply:
Yes, `test_tweakreg.py` is creating `ref_cat.ecsv` in a unit test I moved from the jwst repo.  `test_jump.py` was already creating `data.fits` and `gdq.fits`, which is likely (I didn't test this) happening on main.

Admittedly, adding the .gitignore is a lazy band-aid.  Any suggestions for how to ensure the tests clean up after themselves?